packages/data-transport-layer/src/services/main/service.ts

```diff
@@ -68,6 +68,12 @@ export class L1DataTransportService extends BaseService<L1DataTransportServiceOp
   protected async _init(): Promise<void> {
     this.logger.info('Initializing L1 Data Transport Service...')
 
+    if (this.options.bssHardfork1Index !== null && this.options.bssHardfork1Index !== undefined) {
```

Comment:
Could get the same thing with `typeof this.options.bassHardfork1Index === 'number'` or use `== null` to remove the need to check both